2023-07-10 15:13:34,225 INFO    StreamThr :36957 [internal.py:wandb_internal():86] W&B internal server running at pid: 36957, started at: 2023-07-10 15:13:34.225372
2023-07-10 15:13:34,229 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status
2023-07-10 15:13:34,233 INFO    WriterThread:36957 [datastore.py:open_for_write():85] open: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_151334-1zza2zh5/run-1zza2zh5.wandb
2023-07-10 15:13:34,235 DEBUG   SenderThread:36957 [sender.py:send():369] send: header
2023-07-10 15:13:34,235 DEBUG   SenderThread:36957 [sender.py:send():369] send: run
2023-07-10 15:13:35,371 INFO    SenderThread:36957 [dir_watcher.py:__init__():211] watching files in: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_151334-1zza2zh5/files
2023-07-10 15:13:35,371 INFO    SenderThread:36957 [sender.py:_start_run_threads():1100] run started: 1zza2zh5 with start time 1688973214.225116
2023-07-10 15:13:35,371 DEBUG   SenderThread:36957 [sender.py:send_request():396] send_request: summary_record
2023-07-10 15:13:35,372 INFO    SenderThread:36957 [sender.py:_save_file():1354] saving file wandb-summary.json with policy end
2023-07-10 15:13:35,378 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: check_version
2023-07-10 15:13:35,378 DEBUG   SenderThread:36957 [sender.py:send_request():396] send_request: check_version
2023-07-10 15:13:35,564 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: run_start
2023-07-10 15:13:35,569 DEBUG   HandlerThread:36957 [system_info.py:__init__():31] System info init
2023-07-10 15:13:35,569 DEBUG   HandlerThread:36957 [system_info.py:__init__():46] System info init done
2023-07-10 15:13:35,569 INFO    HandlerThread:36957 [system_monitor.py:start():181] Starting system monitor
2023-07-10 15:13:35,570 INFO    HandlerThread:36957 [system_monitor.py:probe():201] Collecting system info
2023-07-10 15:13:35,570 INFO    SystemMonitor:36957 [system_monitor.py:_start():145] Starting system asset monitoring threads
2023-07-10 15:13:35,573 INFO    SystemMonitor:36957 [interfaces.py:start():190] Started cpu monitoring
2023-07-10 15:13:35,575 INFO    SystemMonitor:36957 [interfaces.py:start():190] Started disk monitoring
2023-07-10 15:13:35,576 INFO    SystemMonitor:36957 [interfaces.py:start():190] Started gpu monitoring
2023-07-10 15:13:35,577 INFO    SystemMonitor:36957 [interfaces.py:start():190] Started memory monitoring
2023-07-10 15:13:35,577 INFO    SystemMonitor:36957 [interfaces.py:start():190] Started network monitoring
2023-07-10 15:13:35,588 DEBUG   HandlerThread:36957 [system_info.py:probe():195] Probing system
2023-07-10 15:13:35,591 DEBUG   HandlerThread:36957 [gitlib.py:_init_repo():53] git repository is invalid
2023-07-10 15:13:35,591 DEBUG   HandlerThread:36957 [system_info.py:probe():240] Probing system done
2023-07-10 15:13:35,591 DEBUG   HandlerThread:36957 [system_monitor.py:probe():210] {'os': 'Linux-5.19.0-46-generic-x86_64-with-glibc2.35', 'python': '3.9.16', 'heartbeatAt': '2023-07-10T07:13:35.588248', 'startedAt': '2023-07-10T07:13:34.213973', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '/home/zhuhe/HPE-with-Diffusion/main.py', 'codePath': 'main.py', 'host': '415-2', 'username': 'zhuhe', 'executable': '/home/zhuhe/.conda/envs/diffhpe/bin/python', 'cpu_count': 16, 'cpu_count_logical': 24, 'cpu_freq': {'current': 4097.904208333333, 'min': 800.0, 'max': 4783.333333333333}, 'cpu_freq_per_core': [{'current': 5100.0, 'min': 800.0, 'max': 5100.0}, {'current': 5103.881, 'min': 800.0, 'max': 5100.0}, {'current': 5099.861, 'min': 800.0, 'max': 5100.0}, {'current': 5100.0, 'min': 800.0, 'max': 5100.0}, {'current': 5100.0, 'min': 800.0, 'max': 5100.0}, {'current': 5103.381, 'min': 800.0, 'max': 5100.0}, {'current': 5100.0, 'min': 800.0, 'max': 5100.0}, {'current': 5100.0, 'min': 800.0, 'max': 5100.0}, {'current': 5031.383, 'min': 800.0, 'max': 5200.0}, {'current': 4902.557, 'min': 800.0, 'max': 5200.0}, {'current': 5043.722, 'min': 800.0, 'max': 5200.0}, {'current': 4992.089, 'min': 800.0, 'max': 5200.0}, {'current': 5100.0, 'min': 800.0, 'max': 5100.0}, {'current': 2100.0, 'min': 800.0, 'max': 5100.0}, {'current': 5100.0, 'min': 800.0, 'max': 5100.0}, {'current': 5101.182, 'min': 800.0, 'max': 5100.0}, {'current': 3700.427, 'min': 800.0, 'max': 4100.0}, {'current': 3871.218, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}, {'current': 2100.0, 'min': 800.0, 'max': 4100.0}], 'disk': {'total': 93.31179428100586, 'used': 0.017917633056640625}, 'gpu': 'NVIDIA GeForce RTX 3090', 'gpu_count': 2, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 3090', 'memory_total': 25769803776}, {'name': 'NVIDIA GeForce RTX 3090', 'memory_total': 25769803776}], 'memory': {'total': 31.185562133789062}}
2023-07-10 15:13:35,591 INFO    HandlerThread:36957 [system_monitor.py:probe():211] Finished collecting system info
2023-07-10 15:13:35,591 INFO    HandlerThread:36957 [system_monitor.py:probe():214] Publishing system info
2023-07-10 15:13:35,592 DEBUG   HandlerThread:36957 [system_info.py:_save_pip():51] Saving list of pip packages installed into the current environment
2023-07-10 15:13:35,592 DEBUG   HandlerThread:36957 [system_info.py:_save_pip():67] Saving pip packages done
2023-07-10 15:13:35,592 DEBUG   HandlerThread:36957 [system_info.py:_save_conda():74] Saving list of conda packages installed into the current environment
2023-07-10 15:13:36,374 INFO    Thread-17 :36957 [dir_watcher.py:_on_file_created():272] file/dir created: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_151334-1zza2zh5/files/requirements.txt
2023-07-10 15:13:36,374 INFO    Thread-17 :36957 [dir_watcher.py:_on_file_created():272] file/dir created: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_151334-1zza2zh5/files/wandb-summary.json
2023-07-10 15:13:36,374 INFO    Thread-17 :36957 [dir_watcher.py:_on_file_created():272] file/dir created: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_151334-1zza2zh5/files/conda-environment.yaml
2023-07-10 15:13:36,510 DEBUG   HandlerThread:36957 [system_info.py:_save_conda():86] Saving conda packages done
2023-07-10 15:13:36,512 INFO    HandlerThread:36957 [system_monitor.py:probe():216] Finished publishing system info
2023-07-10 15:13:36,514 DEBUG   SenderThread:36957 [sender.py:send():369] send: files
2023-07-10 15:13:36,515 INFO    SenderThread:36957 [sender.py:_save_file():1354] saving file wandb-metadata.json with policy now
2023-07-10 15:13:36,521 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 15:13:36,522 DEBUG   SenderThread:36957 [sender.py:send_request():396] send_request: stop_status
2023-07-10 15:13:37,032 ERROR   wandb-upload_0:36957 [internal_api.py:upload_file():2192] upload_file exception https://storage.googleapis.com/wandb-production.appspot.com/ada4321/DiffHPE/1zza2zh5/wandb-metadata.json?Expires=1689059616&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=ts5QbwZ39Z0BiI9PUCdaZHi6MOrYVC3BpBcB2VwLb0PaZ0idSIPEpXNN%2BWu%2F8wkJM2LxWn8swoCfyWInfpovELFytVw3A3ufQim9Hc9LZP13oZ689Ua1UNjHUMtNkspdSKcEojRN1MvfNvUnMLVBig2hhwRP243JV5tPdnyUmMv5cXVULfC90ulqj7ma%2FrrZZDECtFkU%2FXE55HBOosOKmznwyIPTR2PhrfbSH9n1P9cuBGQ0Nu9OW%2FCDEMlOMTDCIn5f%2Fuug6VgFp9eBuQB0%2FVj7eQlmaT%2B7yxqjmtTDkE1GeYm4LWdjy6Pla%2Fz3nF41SHZrbQERhFjPf0Mulp2qxQ%3D%3D: 404 Client Error: Not Found for url: https://storage.googleapis.com/wandb-production.appspot.com/ada4321/DiffHPE/1zza2zh5/wandb-metadata.json?Expires=1689059616&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=ts5QbwZ39Z0BiI9PUCdaZHi6MOrYVC3BpBcB2VwLb0PaZ0idSIPEpXNN%2BWu%2F8wkJM2LxWn8swoCfyWInfpovELFytVw3A3ufQim9Hc9LZP13oZ689Ua1UNjHUMtNkspdSKcEojRN1MvfNvUnMLVBig2hhwRP243JV5tPdnyUmMv5cXVULfC90ulqj7ma%2FrrZZDECtFkU%2FXE55HBOosOKmznwyIPTR2PhrfbSH9n1P9cuBGQ0Nu9OW%2FCDEMlOMTDCIn5f%2Fuug6VgFp9eBuQB0%2FVj7eQlmaT%2B7yxqjmtTDkE1GeYm4LWdjy6Pla%2Fz3nF41SHZrbQERhFjPf0Mulp2qxQ%3D%3D
2023-07-10 15:13:37,032 ERROR   wandb-upload_0:36957 [internal_api.py:upload_file():2194] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '3625'}
2023-07-10 15:13:37,032 ERROR   wandb-upload_0:36957 [internal_api.py:upload_file():2196] upload_file response body: b'<!DOCTYPE html>\n<html lang=en>\n  <meta charset=utf-8>\n  <meta name=viewport content="initial-scale=1, minimum-scale=1, width=device-width">\n  <title>Error 404 (Not Found)!!1</title>\n  <style>\n    *{margin:0;padding:0}html,code{font:15px/22px arial,sans-serif}html{background:#fff;color:#222;padding:15px}body{margin:7% auto 0;max-width:390px;min-height:180px;padding:30px 0 15px}* > body{background:url(//www.google.com/images/errors/robot.png) 100% 5px no-repeat;padding-right:205px}p{margin:11px 0 22px;overflow:hidden}ins{color:#777;text-decoration:none}a img{border:0}@media screen and (max-width:772px){body{background:none;margin-top:0;max-width:none;padding-right:0}}#logo{background:url(//www.google.com/images/branding/googlelogo/1x/googlelogo_color_150x54dp.png) no-repeat;margin-left:-5px}@media only screen and (min-resolution:192dpi){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat 0% 0%/100% 100%;-moz-border-image:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) 0}}@media only screen and (-webkit-min-device-pixel-ratio:2){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat;-webkit-background-size:100% 100%}}#logo{display:inline-block;height:54px;width:150px}\n  </style>\n  <a href=//www.google.com/><span id=logo aria-label=Google></span></a>\n  <p><b>404.</b> <ins>That\xe2\x80\x99s an error.</ins>\n  <p>The requested URL <code>/wandb-production.appspot.com/ada4321/DiffHPE/1zza2zh5/wandb-metadata.json</code> was not found on this server.  <ins>That\xe2\x80\x99s all we know.</ins>\n'
2023-07-10 15:13:37,314 DEBUG   SenderThread:36957 [sender.py:send():369] send: telemetry
2023-07-10 15:13:37,372 INFO    Thread-17 :36957 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_151334-1zza2zh5/files/conda-environment.yaml
2023-07-10 15:13:37,374 INFO    Thread-17 :36957 [dir_watcher.py:_on_file_created():272] file/dir created: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_151334-1zza2zh5/files/wandb-metadata.json
2023-07-10 15:13:37,375 INFO    Thread-17 :36957 [dir_watcher.py:_on_file_created():272] file/dir created: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_151334-1zza2zh5/files/output.log
2023-07-10 15:13:37,610 ERROR   wandb-upload_0:36957 [upload_job.py:push():136] Failed to upload file: /tmp/tmpdjytkbqzwandb/b6a1y78c-wandb-metadata.json
Traceback (most recent call last):
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 212, in wrapped_fn
    return retrier(*args, **kargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2218, in upload_file
    wandb._sentry.reraise(e)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/analytics/sentry.py", line 146, in reraise
    raise exc.with_traceback(sys.exc_info()[2])
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2190, in upload_file
    response.raise_for_status()
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://storage.googleapis.com/wandb-production.appspot.com/ada4321/DiffHPE/1zza2zh5/wandb-metadata.json?Expires=1689059616&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=ts5QbwZ39Z0BiI9PUCdaZHi6MOrYVC3BpBcB2VwLb0PaZ0idSIPEpXNN%2BWu%2F8wkJM2LxWn8swoCfyWInfpovELFytVw3A3ufQim9Hc9LZP13oZ689Ua1UNjHUMtNkspdSKcEojRN1MvfNvUnMLVBig2hhwRP243JV5tPdnyUmMv5cXVULfC90ulqj7ma%2FrrZZDECtFkU%2FXE55HBOosOKmznwyIPTR2PhrfbSH9n1P9cuBGQ0Nu9OW%2FCDEMlOMTDCIn5f%2Fuug6VgFp9eBuQB0%2FVj7eQlmaT%2B7yxqjmtTDkE1GeYm4LWdjy6Pla%2Fz3nF41SHZrbQERhFjPf0Mulp2qxQ%3D%3D

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/filesync/upload_job.py", line 127, in push
    self._api.upload_file_retry(
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/apis/normalize.py", line 51, in wrapper
    raise CommError(message, error)
wandb.errors.CommError: <Response [404]>
2023-07-10 15:13:37,840 ERROR   Thread-16 :36957 [step_upload.py:_handle_event():200] Failed to upload file: /tmp/tmpdjytkbqzwandb/b6a1y78c-wandb-metadata.json
Traceback (most recent call last):
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 212, in wrapped_fn
    return retrier(*args, **kargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2218, in upload_file
    wandb._sentry.reraise(e)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/analytics/sentry.py", line 146, in reraise
    raise exc.with_traceback(sys.exc_info()[2])
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/sdk/internal/internal_api.py", line 2190, in upload_file
    response.raise_for_status()
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://storage.googleapis.com/wandb-production.appspot.com/ada4321/DiffHPE/1zza2zh5/wandb-metadata.json?Expires=1689059616&GoogleAccessId=gorilla-files-url-signer-man%40wandb-production.iam.gserviceaccount.com&Signature=ts5QbwZ39Z0BiI9PUCdaZHi6MOrYVC3BpBcB2VwLb0PaZ0idSIPEpXNN%2BWu%2F8wkJM2LxWn8swoCfyWInfpovELFytVw3A3ufQim9Hc9LZP13oZ689Ua1UNjHUMtNkspdSKcEojRN1MvfNvUnMLVBig2hhwRP243JV5tPdnyUmMv5cXVULfC90ulqj7ma%2FrrZZDECtFkU%2FXE55HBOosOKmznwyIPTR2PhrfbSH9n1P9cuBGQ0Nu9OW%2FCDEMlOMTDCIn5f%2Fuug6VgFp9eBuQB0%2FVj7eQlmaT%2B7yxqjmtTDkE1GeYm4LWdjy6Pla%2Fz3nF41SHZrbQERhFjPf0Mulp2qxQ%3D%3D

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/filesync/step_upload.py", line 288, in run_and_notify
    self._do_upload_sync(event)
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/filesync/step_upload.py", line 330, in _do_upload_sync
    job.run()
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/filesync/upload_job.py", line 57, in run
    self.push()
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/filesync/upload_job.py", line 127, in push
    self._api.upload_file_retry(
  File "/home/zhuhe/.conda/envs/diffhpe/lib/python3.9/site-packages/wandb/apis/normalize.py", line 51, in wrapper
    raise CommError(message, error)
wandb.errors.CommError: <Response [404]>
2023-07-10 15:13:39,317 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:13:39,373 INFO    Thread-17 :36957 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_151334-1zza2zh5/files/output.log
2023-07-10 15:13:44,318 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:13:49,319 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:13:51,521 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 15:13:51,521 DEBUG   SenderThread:36957 [sender.py:send_request():396] send_request: stop_status
2023-07-10 15:13:54,821 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:13:59,822 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:14:04,828 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:14:05,381 INFO    Thread-17 :36957 [dir_watcher.py:_on_file_modified():289] file/dir modified: /home/zhuhe/HPE-with-Diffusion/wandb/run-20230710_151334-1zza2zh5/files/config.yaml
2023-07-10 15:14:06,521 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 15:14:06,522 DEBUG   SenderThread:36957 [sender.py:send_request():396] send_request: stop_status
2023-07-10 15:14:10,830 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:14:15,831 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:14:20,832 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:14:21,521 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 15:14:21,522 DEBUG   SenderThread:36957 [sender.py:send_request():396] send_request: stop_status
2023-07-10 15:14:25,832 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:14:30,833 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:14:35,578 DEBUG   SystemMonitor:36957 [system_monitor.py:_start():159] Starting system metrics aggregation loop
2023-07-10 15:14:35,580 DEBUG   SenderThread:36957 [sender.py:send():369] send: stats
2023-07-10 15:14:36,521 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 15:14:36,522 DEBUG   SenderThread:36957 [sender.py:send_request():396] send_request: stop_status
2023-07-10 15:14:36,841 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:14:41,842 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:14:46,843 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:14:51,522 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 15:14:51,522 DEBUG   SenderThread:36957 [sender.py:send_request():396] send_request: stop_status
2023-07-10 15:14:51,848 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:14:56,849 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:15:01,850 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:15:05,581 DEBUG   SenderThread:36957 [sender.py:send():369] send: stats
2023-07-10 15:15:06,522 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 15:15:06,522 DEBUG   SenderThread:36957 [sender.py:send_request():396] send_request: stop_status
2023-07-10 15:15:07,804 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:15:12,805 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:15:17,806 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:15:21,522 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 15:15:21,523 DEBUG   SenderThread:36957 [sender.py:send_request():396] send_request: stop_status
2023-07-10 15:15:22,848 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:15:27,849 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:15:32,851 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:15:35,583 DEBUG   SenderThread:36957 [sender.py:send():369] send: stats
2023-07-10 15:15:36,522 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 15:15:36,523 DEBUG   SenderThread:36957 [sender.py:send_request():396] send_request: stop_status
2023-07-10 15:15:38,805 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:15:43,806 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:15:48,806 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:15:51,522 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 15:15:51,523 DEBUG   SenderThread:36957 [sender.py:send_request():396] send_request: stop_status
2023-07-10 15:15:53,852 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:15:58,853 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:16:03,854 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:16:05,584 DEBUG   SenderThread:36957 [sender.py:send():369] send: stats
2023-07-10 15:16:06,522 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: stop_status
2023-07-10 15:16:06,523 DEBUG   SenderThread:36957 [sender.py:send_request():396] send_request: stop_status
2023-07-10 15:16:09,764 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
2023-07-10 15:16:14,765 DEBUG   HandlerThread:36957 [handler.py:handle_request():144] handle_request: status_report
